rpm/SPECS/couchdb.spec

```diff
@@ -82,32 +102,47 @@ languages and environments.
 
 %build
 ./configure -c
-%{__make} release
+%{__make} release %{?_smp_mflags}
```

Comment:
No, our make process will not tolerate additional flags being added for smp. Kindly remove this.

Reply:
I will change this too.

Reply:
But what's about the original version of `js/rpm/SPECS/js.spec`? There are the same make flags!